GetIntoTeachingApi/Startup.cs

```diff
@@ -162,16 +162,25 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseAuthorization();
 
+            // Configure recurring jobs.
+            RecurringJob.AddOrUpdate<CrmSyncJob>("crm-sync", (x) => x.Run(), Cron.Daily());
+            RecurringJob.AddOrUpdate<LocationSyncJob>("location-sync", (x) => 
+                x.RunAsync("https://www.freemaptools.com/download/full-postcodes/ukpostcodes.zip"), Cron.Weekly());
+
             using (var serviceScope = app.ApplicationServices.CreateScope())
             {

```

Comment:
does this particular 'using' need to be scoped?

Reply:
Nope; ReSharper picks up on them but not that one for some reason 🤷‍♂️